teamspeak3/teamspeak3.json

```diff
@@ -0,0 +1,75 @@
+{
+  "pufferd": {
+    "type": "Voice",
+    "display": "Teamspeak 3",
+    "install": {
+      "commands": [
+        {
+          "files": "http://dl.4players.de/ts/releases/3.1.1/teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
```

Comment:
Should be able to use a ${version} tag if they do it, refer to our Minecraft ones for example.